api/src/main/java/org/openmrs/module/reporting/report/renderer/ReportTemplateRenderer.java

```diff
@@ -228,7 +228,7 @@ public Object getReplacementValue(Object initialValue) {
 				replacementValue = new Integer(((Cohort) initialValue).size());
 			} 
 			else if (initialValue instanceof IndicatorResult) {
-				replacementValue = new Double(((IndicatorResult) initialValue).getValue().doubleValue());
+				replacementValue = new Integer((Integer) ((IndicatorResult) initialValue).getValue());

```

Comment:
@solemabrothers What will you do for values which have decimal points in some indicators

Reply:
@solemabrothers you could try using the steps here https://stackoverflow.com/a/25308216/200120 to format the decimals